### PR TITLE
fix(FEC-12189): Stream doesn't start if open a program with a bookmark

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEC-12189

related pr: 
https://github.com/kaltura/playkit-js/pull/652

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/dash-adapter.js [hunk 1]
```diff
@@ -243,7 +243,11 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     }
     if (Utils.Object.hasPropertyPath(config, 'streaming')) {
       adapterConfig.forceBreakStall = Utils.Object.getPropertyPath(config, 'streaming.forceBreakStall');
-      adapterConfig.shakaConfig.streaming.lowLatencyMode = Utils.Object.getPropertyPath(config, 'streaming.lowLatencyMode');
+      if (typeof Utils.Object.getPropertyPath(config, 'streaming.lowLatencyMode') === 'boolean') {
+        adapterConfig.shakaConfig.streaming.lowLatencyMode = Utils.Object.getPropertyPath(config, 'streaming.lowLatencyMode');
+      } else {
+        adapterConfig.shakaConfig.streaming.lowLatencyMode = this.isLive() ? true : false;
```
Comment (yairans) [inline src/dash-adapter.js:249]: you can't know if it's live or not at that moment, move it after the media loading

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/dash-adapter.js (hunk 1 above)
Comment (JonathanTGold) [reply src/dash-adapter.js]: done

**Review by SivanA-Kaltura — APPROVED** [review]

**Review by yairans — APPROVED** [review]

File: test/src/dash-adapter.spec.js [hunk 2]
```diff
@@ -213,6 +213,38 @@ describe('DashAdapter: load', () => {
       });
   });
 
+  it('should set streaming.lowLatencyMod config to false on vod by default', done => {
```
Comment (SivanA-Kaltura) [inline test/src/dash-adapter.spec.js:216]:
```suggestion
  it('should set streaming.lowLatencyMode config to false on vod by default', done => {
```

File: test/src/dash-adapter.spec.js [hunk 3]
```diff
@@ -213,6 +213,38 @@ describe('DashAdapter: load', () => {
...
+
+      dashInstance.load().then(() => {
+        video.play();
+      });
+    } catch (e) {
+      done(e);
+    }
+  });
+
+  it('should set streaming.lowLatencyMod config to true on live by default', done => {
```
Comment (SivanA-Kaltura) [inline test/src/dash-adapter.spec.js:232]:
```suggestion
  it('should set streaming.lowLatencyMode config to true on live by default', done => {
```

File: test/src/dash-adapter.spec.js [hunk 4]
```diff
@@ -213,6 +213,38 @@ describe('DashAdapter: load', () => {
...
+
+      dashInstance.load().then(() => {
+        video.play();
+      });
+    } catch (e) {
+      done(e);
+    }
+  });
+
   it('should load successfully when given a valid video to play', done => {
```
Comment (SivanA-Kaltura) [inline test/src/dash-adapter.spec.js:248]: we should add a test (or update an existing test) to check the use cases of setting lowLatencyMode manually in config (VOD source + set to true in config = true, Live source + set to false in config = false)

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: test/src/dash-adapter.spec.js (hunk 2 above)
Comment (giladna) [inline test/src/dash-adapter.spec.js:216]: Change Mod  to Mode

**Review by giladna — APPROVED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**